README.md

```diff
@@ -16,212 +16,42 @@ This is a beta build and is currently under active development. Please be advise
-
-#### Build Types
-
-| Type  | Purpose | Stability | Audience | Identifier | Example Version |
-| :---- | :--------- | :---------- | :-------- | :------- | :--- |
-| **alpha** | **Sandbox.** For developers to verify behavior and try features. Things seen here might never go to production. Most bugs here can be ignored.| Unstable: Expect bugs | Internal developers | 0XX | 1.2.3-alpha04 (10203004) |
-| **beta** | **Hand-off.** For developers to present finished features. Bugs found here should be reported and immediately addressed, if they relate to recent changes. | Unstable: Report bugs | Internal stakeholders | 2XX | 1.2.3-beta04 (10203204) |
-| **release candidate** | **Hardening.** Final testing for an app release that we believe is ready to go live. The focus here is regression testing to ensure that new changes have not introduced instability in areas that were previously working.  | Stable: Hunt for bugs | External testers | 4XX | 1.2.3-rc04 (10203404) |
-| **production** | **Delivery.** Deliver new features to end-users. Any bugs found here need to be prioritized. Some will require immediate attention but most can be worked into a future release. | Stable: Prioritize bugs | Public | 8XX | 1.2.3 (10203800) |
-
-[Back to contents](#contents)
-
-## Examples
+# Consumers
+If you're a developer consuming this SDK in your own app, see [Consumers.md](docs/Consumers.md) for a discussion of setting up your app to consume the SDK and leverage the public APIs.
 
 A primitive example to exercise the SDK exists in this repo, under [Demo App](demo-app).
 
-There's also a more comprehensive [Sample Wallet](https://github.com/zcash/zcash-android-wallet).
-
-[Back to contents](#contents)
-
-## Checkpoints
-To improve the speed of syncing with the Zcash network, the SDK contains a series of embedded checkpoints.  These should be updated periodically, as new transactions are added to the network.  Checkpoints are stored under the [assets](sdk-lib/src/main/assets/co.electriccoin.zcash/checkpoint) directory as JSON files.  Checkpoints for both mainnet and testnet are bundled into the SDK.
-
-To update the checkpoints, see [Checkmate](https://github.com/zcash-hackworks/checkmate).
-
-We generally recommend adding new checkpoints every few weeks.  By convention, checkpoints are added in block increments of 10,000 which provides a reasonable tradeoff in terms of number of checkpoints versus performance.
-
-There are two special checkpoints, one for sapling activation and another for orchard activation.  These are mentioned because they don't follow the "round 10,000" rule.
- * Sapling activation
-     * Mainnet: 419200
-     * Testnet: 280000
- * Orchard activation
-     * Mainnet: 1687104
-     * Testnet: 1842420
-
-## Publishing
+There are also more comprehensive sample walletes:
```

Comment:
```suggestion
There are also more comprehensive sample wallets:
```

Reply:
It's French for `wallets` 😂.

This is a pre-existing issue in the main branch.

Reply:
That's a typo in the `main` branch; it should be fixed there.